tools/121-platform-https.conf

```diff
@@ -240,6 +240,7 @@ ServerSignature Off
   # Legacy support
   #
   Redirect permanent /HO-portal /portal
+  Redirect permanent /PA-app /app
```

Comment:
Note to future-self: This needs to be added AFTER the "upgrade legacy urls"-period.

So that the server-specific config can catch anything more specific, otherwise the users will already have been redirected by this line before any server-specific rules would start to apply.

Reply:
this change was reverted in the end. Instead put in the .env.conf file.